src/view/components/__tests__/CountryDropdown.test.tsx

```diff
@@ -0,0 +1,19 @@
+import "@testing-library/jest-dom";
+import { render } from "@testing-library/react";
+import React from "react";
+import { CountryDropdown } from "../countries/CountryDropdown";
+
+describe("CountryDropdown", () => {
+    it("Dropdown button should show add a country", () => {
```

Comment:
are you able to change the string to this? `it("should show add a country", ...) the string inside the it function is written as if "it" is the first word